Add a custom capability 'cloud:URL' and relevant documentation (#20)
testproject-io/java-sdk: README.md

It can also be set using the `TP_AGENT_URL` environment variable.

## Remote (Cloud) Driver

> By default, TestProject Agent is using local Selenium and Appium servers. \

Reviewer: ```suggestion
By default, TestProject Agent communicates with the local Selenium or Appium server. \
```

Author: Fixed.
